src/components/tools/image-tool.tsx

```diff
@@ -265,7 +265,7 @@ export default class ImageToolComponent extends BaseComponent<IProps, IState> {
       gImageMap.getImage(newUrl)
         .then(image => {
           if (image.contentUrl && !isPlaceholderImage(image.displayUrl)) {
-            this.getContent().setUrl(image.contentUrl);
+            this.getContent().url = image.contentUrl;
```

Comment:
I might be reading this wrong. But getContent returns an MST model, you should not be setting the property directly. You should use an action. MST will show an error if you do this, so if I have this right then, this bit of code hasn't been tested.